operational/azure/byol_report/azure_byol_report.pt

```diff
@@ -542,3 +573,93 @@ escalation "esc_email" do
   description "Send incident email"
   email $param_email
 end
+
```

Comment:
This policy template has a Meta Policy section but no meta parent policy template. Is this intentional (and you are just future proofing)?

Reply:
Future proofing. It's possible that, if the user has enough data, we might need to make a meta version to avoid time outs. Doing so would require adding logic to the meta policy script to handle the lack of an Azure credential, though.